Defly.io/dist/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "author": {
+    "name": "Apodemus",
+    "id": "223817321695674371"
+  },
+  "url": "defly.io",
+  "description": {
+    "en": "Fly your helicopter and shoot at enemies. Build walls to protect yourself and conquer as much territory as possible."
+  },
+  "service": "Defly.io",
+  "logo": "https://defly.io/favicon/android-icon-192x192.png",
+  "thumbnail": "https://defly.io/favicon/android-icon-192x192.png",
+  "color": "#3D5DFF",
+  "tags": [],
+  "category": "games"
```

Comment:
you need atleast 1 tag

Reply:
tags are now: `["game", "defly", "multiplayer", "defly.io", "browser game"]`